### ISSUE TITLE
unable to delete metrics

### ISSUE THREAD

**Issue by jefframin** [performancecopilot/parfait#62]
Unless I am mistaken, there is no mechanism for removing a Monitorable from the MonitorableRegistry. 

Could removal of a Monitorable be implemented?

**Comment by natoscott** [comment]
@jefframin yes, it could be implemented - the underlying PCP mechanisms support this, it would require some Java coding I believe.

**Comment by jefframin** [comment]
If this functionality is ever implemented, will this issue be updated so I would be notified?

Thanks.

**Comment by natoscott** [comment]
@jefframin yep
